core/src/edu/asu/c3simulator/simulation/Employee.java

```diff
@@ -0,0 +1,248 @@
+package edu.asu.c3simulator.simulation;
+
+import java.util.Random;
+
+public class Employee
+{
+	public String employeeName;
+	public String employeePosition;
+	public String randomEmployeePosition;
+	public int employeePay;
+	public int employeePref;
+	public int employeeMorale;
+	public int netSalary;
+	public float averageAnnualRaise;
+	public float averageAnnualBonus;
+	public int netBonuses;
+	
+	public Employee(String name, String position, int pay, int pref, int morale,
+			int net_salary, float annualRaise, float annualBonus, int net_bonus)
```

Comment:
I think a lot of these fields should be calculated, rather than being specified in the constructor.

Was this intended to be used for loading?